WARNING  python29:jks_login.py:25 可能未成功进入页面【: http://jenkins.senguo.me/login?from=%2F】，可能会引发异常!
WARNING  python29:daily_static_page.py:24 可能未成功进入页面【每日统计更新: http://jenkins.senguo.me/view/%E7%BB%9F%E8%AE%A1%E4%BB%BB%E5%8A%A1/job/UTIL-PF%EF%BC%9A%E6%AF%8F%E6%97%A5%E7%BB%9F%E8%AE%A1%E6%9B%B4%E6%96%B0/build?delay=0sec】，可能会引发异常!
INFO     python29:test_statistic.py:24 店铺id2670,days:101
INFO     python29:test_statistic.py:24 店铺id2670,days:100
INFO     python29:test_statistic.py:24 店铺id2670,days:99
INFO     python29:test_statistic.py:24 店铺id2670,days:98
INFO     python29:test_statistic.py:24 店铺id2670,days:97
INFO     python29:test_statistic.py:24 店铺id2670,days:96
INFO     python29:test_statistic.py:24 店铺id2670,days:95
INFO     python29:test_statistic.py:24 店铺id2670,days:94
INFO     python29:test_statistic.py:24 店铺id2670,days:93
INFO     python29:test_statistic.py:24 店铺id2670,days:92
INFO     python29:test_statistic.py:24 店铺id2670,days:91
INFO     python29:test_statistic.py:24 店铺id2670,days:90
INFO     python29:test_statistic.py:24 店铺id2670,days:89
INFO     python29:test_statistic.py:24 店铺id2670,days:88
INFO     python29:test_statistic.py:24 店铺id2670,days:87
INFO     python29:test_statistic.py:24 店铺id2670,days:86
INFO     python29:test_statistic.py:24 店铺id2670,days:85
ERROR    python29:selenium_handler.py:38 元素找不到('xpath', "//input[@value='DAYS']/following-sibling::input[@name='value']")
INFO     python29:conftest.py:31 退出浏览器img...
INFO     python29:conftest.py:33 浏览器退出成功.